lib/user.js

```diff
@@ -39,11 +40,23 @@ class User {
     return this.token;
   }
 
-  login(email, password, cb) {
+  login(email, password, done) {
+    const envEmail = process.env[constants.EnvironmentVariables.User];
+    const envPassword = process.env[constants.EnvironmentVariables.Password];
+    const useEnvVariables = !email && !password && envEmail && envPassword;
```

Comment:
probably don't need to create a variable for this test since it's only used in a single place